### DISCUSSION TITLE
There was an error while loading site data - not found.

### DISCUSSION THREAD

**Discussion by sardarms** [sveltia/sveltia-cms#81] [Q&A]
I am receiving this error. Can someone please help me? 

I am using Hugo. Everything was done and I have used Github to sign in. 

![image](https://github.com/sveltia/sveltia-cms/assets/83923880/b4942f8c-bcf2-4c56-b148-2342372cabef)

**Comment by kyoshino** [comment]
- Was your previous Netlify CMS or Decap CMS instance working before migrating to Sveltia CMS?
- What’s your backend — GitHub or local?

**Reply by sardarms** [reply]
1. Decap CMS was working properly. Then I decided to use sveltia-cms . 
2. Github. I was successfully able to login. But then I got this message.

**Reply by kyoshino** [reply]
Okay. Can you please open the browser’s DevTools ([Chrome](https://developer.chrome.com/docs/devtools/open/), [Firefox](https://developer.mozilla.org/en-US/docs/Learn/Common_questions/Tools_and_setup/What_are_browser_developer_tools), [Safari](https://developer.apple.com/documentation/safari-developer-tools/develop-menu)), load the CMS, take screenshots of the **Console** and **Network** tabs, and paste them here? You can mask confidential parts, e.g. your organization name, if needed.

**Comment by sardarms** [comment]
![image](https://github.com/sveltia/sveltia-cms/assets/83923880/4c3889ad-b294-44db-bd62-9225987953ff)


![image](https://github.com/sveltia/sveltia-cms/assets/83923880/33cc74c7-62e3-4c03-bd0e-df1c53524975)

**Reply by kyoshino** [reply]
Thanks! So you’re using Firefox. Can you reload the page again and see if there is a pink 404 status on the Network tab like this?

<img width="417" alt="Screenshot 2023-11-28 at 3 17 23 AM" src="https://github.com/sveltia/sveltia-cms/assets/2929505/656fb5db-0a23-4523-874d-067ec9324eb3">

**Comment by kyoshino** [comment]
By any chance, are you using GitHub Enterprise Server? I’ve found an issue with the Enterprise support that probably leads to a Not Found error, but not sure if that’s your case.

**Comment by sardarms** [comment]
![image](https://github.com/sveltia/sveltia-cms/assets/83923880/7069782d-e75c-4855-90b0-801bfc83f0c6)


I am not using Github enterprise.

**Reply by kyoshino** [reply]
It might be related to Netlify Identity Widget. Will check.

**Reply by kyoshino** [reply]
Well, maybe unrelated. Actually, there are many reasons [GitHub returns 404 Not Found](https://docs.github.com/en/rest/overview/troubleshooting-the-rest-api#404-not-found-for-an-existing-resource) even if the resource does exist. I’ve just updated Sveltia CMS so you’ll be asked to sign in again. Can you try again to see if it works? Also, can you read the linked doc to see if you have any permission issues?

**Comment by sardarms** [comment]
![image](https://github.com/sveltia/sveltia-cms/assets/83923880/916f48ca-90db-47a6-b5a9-4906e5967860)


![image](https://github.com/sveltia/sveltia-cms/assets/83923880/749c6f75-75c6-4dd3-91ca-c84f132a3ee1)

I still got the same error. 

Thanks for the link on Github 404, I will see if I can do anything.

**Reply by kyoshino** [reply]
`api.github.com/user` returns a 200 success code, meaning you’re successfully authenticated with GitHub and your access token is valid.

`api.github.com/repos/${owner}/${repo}/git/trees/${branch}?recursive=1` retrieves a file list in your repository. A 404 error here probably means your OAuth app (“MSRD Netlify Auth App”) doesn’t have access to the repository (`repo` scope). Not sure...

My OAuth app ([created by myself](https://github.com/sveltia/sveltia-cms-auth)) looks like this:

<img width="667" alt="image" src="https://github.com/sveltia/sveltia-cms/assets/2929505/df9b14b9-6929-45d0-a8f4-e2fef75ffa0a">


Similar discussions: https://github.com/orgs/community/discussions/24604
